hasura/metadata/databases/bdjuno/tables/public_shield_pool.yaml

```diff
@@ -0,0 +1,29 @@
+table:
+  name: shield_pool
+  schema: public
+object_relationships:
+- name: shield_purchase
+  using:
+    manual_configuration:
+      column_mapping:
+        pool_id: pool_id
+      insertion_order: null
+      remote_table:
+        name: shield_purchase
+        schema: public
```

Comment:
seems it could be simplified to array relationship (1 pool_id can have several pool purchases)
```yaml
array_relationships:
- name: shield_purchases
  using:
    foreign_key_constraint_on:
      column: pool_id
      table:
        name: shield_purchase
        schema: public
```